### PR TITLE
:mortar_board: Topic number bases --- Decimal explination

### PR DESCRIPTION
**Pull request by jameshughes89**
### What

Explain decimal numbers

### Why

Although they obviously know them, breaking it down will help with learning other bases since it's the same underlying rules. The problem is, most people don't really think about the underlying rules as they interact with integers more abstractly. 

### How

I tried to be detailed enough to hit the key points, but I was a little loose here and there since, well, they know how to count in base 10. 

### Testing

:+1:

### CODE REVIEW

File: site/topics/numbers/number-bases.rst [hunk 1]
```diff
@@ -2,11 +2,251 @@
 Number Bases
 ************
 
+* Have you ever *really* thought about counting?
```
Comment (twentylemon) [inline site/topics/numbers/number-bases.rst]: Honestly, remove. My initial reaction is just being insulted. I can't think of a non-disparaging intro, so no intro is good enough.

File: site/topics/numbers/number-bases.rst [hunk 2]
```diff
@@ -2,11 +2,251 @@
...
+
 
 
 Decimal (Base Ten)
 ==================
 
+* The decimal number system is used in everyday life
+* The decimal number system is based ten; it consists of ten different symbols/numerals
```
Comment (twentylemon) [inline site/topics/numbers/number-bases.rst]: **base** 10, not based

File: site/topics/numbers/number-bases.rst [hunk 3]
```diff
@@ -2,11 +2,251 @@
...
+
+    * Reset the symbol to zero in the current position
+    * Change to the subsequent symbol in the next position
+
+
+    .. list-table:: Counting with eight digits
+        :widths: 50 50
+
+        * - :math:`00000000`
+          - Zero
```
Comment (twentylemon) [inline site/topics/numbers/number-bases.rst]: Can probably just start counting as `...` then 9, no need to duplicate so much. The prior table is still on the screen.

File: site/topics/numbers/number-bases.rst [hunk 4]
```diff
@@ -2,11 +2,251 @@
...
+          - Millions
+          - :math:`10^{6}`
+          - :math:`1000000`
+        * - Eighth Digit
+          - Ten Millions
+          - :math:`10^{7}`
+          - :math:`10000000`
+
+
+* As a consequence of the counting patter, each position corresponds to a value that is the base raised to some power
```
Comment (twentylemon) [inline site/topics/numbers/number-bases.rst]: typo _pattern_

File: site/topics/numbers/number-bases.rst (hunk 3 above)
Comment (jameshughes89) [reply site/topics/numbers/number-bases.rst]: I'll keep the 8, but yeah.

**Review by twentylemon — APPROVED** [review]

**Pull request merged**